### DISCUSSION TITLE
Is there a plan for the next version that includes ntfy?

### DISCUSSION THREAD

**Discussion by wsw70** [jamesoff/simplemonitor#1375] [Q&A] [answered]
I see that `ntfy` was added in the develop branch and that it even made it to the docs but is not present in 1.12.1.
Are there plans to release a new version soon?

I am asking because I want to weight "running a develop clone" vs "running the proper thing"

**Comment by jamesoff** [comment, marked as answer]
Yeah I can cut a new release - I'm bad at that, I tend to run `develop` (or a feature branch) on my machines and forget to package it up :)

Should really figure out the GitHub Actions stuff to automate that I guess...

**Reply by wsw70** [reply]
Thank you. I will be waiting then for the production version o fmy monitoring.

But while we are at this and since I want to start developing a Home Assistant Logger: how can I run `simplemonitor` from a git clone? It's been some time I have not used Python (but I am back to it) and git bitten by 

```
(...)
    from .Alerters.alerter import Alerter
ImportError: attempted relative import with no known parent package
```

when trying to run `python ./simplemonitor.py`, `python simplemonitor/simplemonitor.py` (from one level up), `python -m simplemonitor.simplemonitor` and some other variations. 

Yes, I am ashamed of myself because there is a Stack Overflow answer with 1700+ upvotes that explains this but I failed.

**Reply by jamesoff** [reply]
No guarantee I don't have the modules arranged weirdly either tbh. Make sure you're on a recent Python, and then I do it by installing in to a virtual environment:

```
# in simplemonitor git directory
python -m venv .venv
source .venv/bin/activate
# optional but recommended, make sure pip is latest
pip install -U pip
pip install -e .
```

Then you should have `simplemonitor` on your path and can just run it. If you want to run it from a service script you can just use the fully path to that inside the `.venv` dir, something like `/home/you/src/simplemonitor/.venv/bin/simplemonitor`. (You may want/need to arrange for the working directory to be right for your config files etc.)

By using `-e` on the pip command, you can just `git pull` in that directory and the updates to the code will apply without you needing to `install` again.
